account_financial_report/tests/test_trial_balance.py

```diff
@@ -444,16 +469,17 @@ def test_03_partner_balance(self):
         )
 
         # Re Generate the trial balance line
-        res_data = self._get_report_lines(with_partners=True)
-        total_amount = res_data["total_amount"]
+        _, res_data = self._get_report_lines(with_partners=True)
+
+        trial_balance = res_data["trial_balance"]
         check_partner_receivable = self.check_partner_in_report(
-            self.account100.id, self.partner.id, total_amount
+            self.account100.id, self.partner.id, trial_balance
```

Comment:
Can you use fixed amounts? These tests are wicked this way.

Reply:
Fixed amounts? This was (and is) trying to pull the corresponding report line filtering by some data (account and partner).